scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
```

Comment:
:stop_sign: **Security Vulnerability**: This code is vulnerable to SQL injection attacks[^4]. Never concatenate user input directly into SQL queries. Use parameterized queries or prepared statements instead.
```suggestion
// Assuming you're using a library like 'mysql2' for database operations
import mysql from 'mysql2/promise'

async function getUserById(id: string) {
    const connection = await mysql.createConnection({
        // connection details
    })
    const [rows] = await connection.execute('SELECT * FROM users WHERE id = ?', [id])
    await connection.end()
    return rows
}
```